spec/controllers/hq/partners_controller_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'will_paginate/array'
```

Comment:
why are we including this in the partner controller spec, I can't see where we are currently using it.